pegasus/sites.v3/code.org/public/learn3/index.haml

```diff
@@ -0,0 +1,102 @@
+---
+title: Learn
+theme: responsive
+social:
+  "og:description": "Anyone can learn the basic concepts of Computer Science. Choose an intro activity to start learning."
+  "og:image": "http://code.org/images/fit-520/codehoc3.jpg"
+  "og:image:width": 520
+  "og:image:height": 391
+  "og:video": ""
+  "og:video:width": ""
+  "og:video:height": ""
+  "og:video:type": ""
+---
+
+= view 'mobilecheck.js'
+
+%script{src: CDO.studio_url('/assets/js/tutorialExplorer.js')}
```

Comment:
Right now we depend on the rails asset pipeline to do stuff like minification, etc. for us. Do we get any of that in pegasus land?